.github/CONTRIBUTING.md

```diff
@@ -371,13 +371,105 @@ This does not go for bug reports however, as we want to unearth and solve as man
 
 ### Commit
 
-* Focus on one thing per commit 
-* Keep changes as small as possible 
-* Only commit related functinoality for each commit 
-* Commit often
-* Strive for each commit to compile/build and pass tests 
-* Use [gitmoji](https://gitmoji.dev) when possible - point to VSCode extension and CLI tool
-* sidenote: when merging the CI pipeline combines each commit to a merge commit containing all commit messages
+You may have encountered statements like
+
+> Commit early, Commit often
+
+We agree. Apart from acting as checkpoints, good commits support collaboration by providing an overview of your changes and make navigating them easy. But only as long the commits are good.
+
+#### The good commit follows Git best practices
+
+You are encouraged to follow commonly adopted Git best practices. We would especially like to highlight:
+
+1. **Try to make each commit a logically separate changeset**
+    - Keep changes in each commit to a minimum
+    - Make changes in each commit _atomic_ &ndash; focus on one thing
+    - Only commit related functionality for each commit
+    - If you have made multiple non-related changes to a file and you need to separate them in several commits, stage your changes using `git add --patch`
+2. **Commit often**
+    - Multiple smaller commits are better than one large commit
+3. **Strive for each commit to compile/build and pass tests**
+4. **Regularly push your commits to remote**
+    - Early peer reviews are easier
+    - In case something happens to your machine it reduces the amount of work lost
+    - Makes it easier to hand over the issue to other developers if needed
+
+#### The good commit has good commit messages
+
+> Keep in mind: [This](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) [has](https://www.git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines) [all](https://github.com/torvalds/subsurface-for-dirk/blob/master/README.md#contributing) [been](http://who-t.blogspot.co.at/2009/12/on-commit-messages.html) [said](https://github.com/erlang/otp/wiki/writing-good-commit-messages) [before](https://github.com/spring-projects/spring-framework/blob/30bce7/CONTRIBUTING.md#format-commit-messages).
```

Comment:
Hah nice - perhaps underline each link to make it obvious that they're individual links.

Reply:
I don't think GitHub markdown styles include underlines. Maybe we should separate the words/links with a non-whitespace character?

Reply:
New commit comming up:

> Keep in mind: This · has · all · been · said · before